### PR TITLE
Create rewards

### PR DESCRIPTION
**Pull request by 7hoenix**

### CODE REVIEW

File: app/models/reward.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Reward < ActiveRecord::Base
+  validates :name, presence: true, uniqueness: true
```
Comment (travishaby) [inline app/models/reward.rb:2]: nice! i forgot to do this using active record and put in some bulky ruby logic to do this--need to fix that on mine

File: app/views/rewards/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>Reward page</h1>
```
Comment (travishaby) [inline app/views/rewards/show.html.erb:1]: Do you need a show page for one reward? Wouldn't an admin just want to see all the rewards with their cost on the index and thats it? Just trying to think about it given a real world context.

File: spec/models/reward_spec.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
```
Comment (travishaby) [inline spec/models/reward_spec.rb:1]: Legit! I haven't made any model tests yet, need to do this!

File: app/views/rewards/show.html.erb (hunk 2 above)
Comment (7hoenix) [inline app/views/rewards/show.html.erb:1]: Great question! I was thinking of rewards as like a full page spread. 

So a user is browsing looking at rewards... and then they click to check one out and get a sense of what their reward would look like. Kind of like Amazon. You are browsing books. And then you click one to see the details.

Do you agree or disagree?

**Pull request merged**